### DISCUSSION TITLE
deploy

### DISCUSSION THREAD

**Discussion by royrajan95** [kalanakt/All-Url-Uploader#72] [General]
Can you please add guide on how to deploy bot on VPS.
Im getting errors...

**Comment by kalanakt** [comment]
Did you follow [Documentation](https://kalanakt.github.io/All-Url-Uploader/)? 

I'll add examples page.

**Reply by royrajan95** [reply]
Don't know about documentation😅.. 
It's working now...
Thanks..

**Comment by kalanakt** [comment]
If available, please add screen shots of your error.

**Reply by code-engineer** [reply]
hey, how do u run this without getting shadowbanned/account terminated by heruko?

**Reply by kalanakt** [reply]
you can deploy it loccaly. read documentation for more details

**Comment by royrajan95** [comment]
hi,
how to get document mode..?

**Reply by kalanakt** [reply]
hi, 
I don't understand your question perfectly
> _Are you looking for documentation: [Click Here](https://kalanakt.github.io/All-Url-Uploader/)_

**Reply by royrajan95** [reply]
I mean, file upload mode : **document mode** / video mode.
